server/src/Server.cpp

```diff
@@ -43,6 +49,12 @@ void Server::run() {
         for(auto session: m_all_client) {
             session->update();
         }
+
+        int sleeping_time = MINIMUM_DT - dt_clock.restart().asMilliseconds();
+
+        if(sleeping_time > 0){
+            usleep(sf::milliseconds(sleeping_time).asMicroseconds());
```

Comment:
usleep does not exist on windows
I suggest a more portable approach using c++ stl

```
#include <chrono>
#include <thread>
...
std::this_thread::sleep_for(std::chrono::microseconds(usec));
```